dart/stalin_sort.dart

```diff
@@ -0,0 +1,14 @@
+List<T> stalinSort<T extends num>(List<T> list) {
+  if (list.isEmpty) return <T>[];
+
+  var x = list.first;
+  return list
+    ..removeWhere((value) {
+      return (x = value >= x ? value : x) != value;
+    });
+}
+
+main() {
```

Comment:
For pattern matters, could you remove the main function and put an exporting function?

Reply:
I fixed to an exporting function only.